Assets/Dev/Si Wai/PermanentPlayerStatistic.cs

```diff
@@ -0,0 +1,106 @@
+
+using UnityEngine;
+[System.Serializable]
+public class PermanentPlayerStatistic : SaveSystem
+{
+    public PermanentStatistic Speed = new();
+    public PermanentStatistic JumpForce = new();
+    public PermanentStatistic MaxHealth = new();
+    private float health;
+    public float Health {
+        get => health;
+        set => health = value > 0 ? value : 0;
+    }
+    private int crumbs;
+    public int Crumbs
+    {
+        get => crumbs;
+        set => crumbs = value > 0 ? value : 0;
+    }
```

Comment:
This however is the only acutal absolute value that can be here. Since this should not have a CurrentStat attacked to it.
Or we find something differnt in the future. but yea, this one can stay
(reaction on the above comment aswell)